hub/core/chunk/base_chunk.py

```diff
@@ -267,7 +267,9 @@ def convert_to_rgb(self, shape):
         return shape
 
     def can_fit_sample(self, sample_nbytes, buffer_nbytes=0):
-        return self.num_data_bytes + buffer_nbytes + sample_nbytes < self.min_chunk_size
+        return (
```

Comment:
minor flaw that I found in our chunking logic, was storing two 8 MB samples in 2 chunks instead of 1 16MB chunk